it/uk/gov/hmrc/fileupload/support/FileActions.scala

```diff
@@ -1,13 +1,27 @@
 package uk.gov.hmrc.fileupload.support
 
 import org.scalatest.Suite
-import org.scalatest.time.{Millis, Seconds, Span}
 import play.api.libs.ws.WSResponse
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId}
 
 trait FileActions extends ActionsSupport {
   this: Suite =>
 
+  val file: String = "-----011000010111000001101001\r\n" +
```

Comment:
refactor it to be as tidy as this:
https://gist.github.com/EECOLOR/7649144

Reply:
done